src/units.jl

```diff
@@ -6,27 +6,30 @@ import ..DEFAULT_QUANTITY_TYPE
 
 @assert DEFAULT_VALUE_TYPE == Float64 "`units.jl` must be updated to support a different default value type."
 
-const _UNIT_SYMBOLS = Symbol[]
-const _UNIT_VALUES = DEFAULT_QUANTITY_TYPE[]
+const UNIT_SYMBOLS = Symbol[]
+const UNIT_VALUES = DEFAULT_QUANTITY_TYPE[]
+const UNIT_MAPPING = Dict{Symbol,Int}()
```

Comment:
It can be defined below?
```suggestion
```

Reply:
Doesn’t even need to be a set, you can just check existence within the `push!` and throw an error if the same symbol was pushed twice

Reply:
I've added `DynamicQuantitiesCollection` with restricted set of operations.